howto/publish-topic/README.md

```diff
@@ -26,21 +26,25 @@ spec:
     value: localhost:6379
   - name: redisPassword
     value: ""
+  - name: allowedTopics
+    value: "deathStartStatus"
 ```
 
+Using the `allowedTopics` you can specify that only the `deathStartStatus` topic should be supported.
+
 To deploy this into a Kubernetes cluster, fill in the `metadata` connection details in the yaml, and run `kubectl apply -f pubsub.yaml`.
 
 ## Publish a topic
 
 To publish a message to a topic, invoke the following endpoint on a Dapr instance:
 
 ```bash
-curl -X POST http://localhost:3500/v1.0/publish/pubsubName/deathStarStatus \
+curl -X POST http://localhost:3500/v1.0/publish/pubsub-name/deathStarStatus \
  -H "Content-Type: application/json" \
  -d '{
       "status": "completed"
     }'
 ```
 
 The above example publishes a JSON payload to a `deathStartStatus` topic.
-Dapr will wrap the user payload in a Cloud Events v1.0 compliant envelope.
+Dapr wraps the user payload in a Cloud Events v1.0 compliant envelope.
```

Comment:
if it is not in Cloud Events 1.0 format, otherwise it passes along an already formatted message.